docs/DeveloperGuide.md

```diff
@@ -227,10 +227,48 @@ Extensions:
 }
 ```
 
+```javascript
+
+{
+    "id": 1,
+    "title": "There's a new DEV theme in town for all you 10x hackers out there (plus one actually useful new feature)",
+    "description": "There's a new DEV theme in town for all you 10x hackers out there (plus one actually useful new feature)",
+    "tags": "meta, changelog, css, ux",
+    "slug": "there-s-a-new-dev-theme-in-town-for-all-you-10x-hackers-out-there-plus-one-actually-useful-new-feature-2kgk",
+    "path": "/devteam/there-s-a-new-dev-theme-in-town-for-all-you-10x-hackers-out-there-plus-one-actually-useful-new-feature-2kgk",
+    "url": "https://dev.to/devteam/there-s-a-new-dev-theme-in-town-for-all-you-10x-hackers-out-there-plus-one-actually-useful-new-feature-2kgk",
+    "canonical_url": "https://dev.to/devteam/there-s-a-new-dev-theme-in-town-for-all-you-10x-hackers-out-there-plus-one-actually-useful-new-feature-2kgk",
+    "comments_count": 37,
+    "positive_reactions_count": 12,
+    "created_at": "2019-10-24T13:41:29Z",
+    "edited_at": "2019-10-24T13:56:35Z",
+    "published_at": "2019-10-24T13:52:17Z",
+    "last_comment_at": "2019-10-25T08:12:43Z",
+    "user": {
+        "name": "Ben Halpern",
+        "username": "ben",
+        "twitter_username": "bendhalpern",
+        "github_username": "benhalpern",
+        "website_url": "http://benhalpern.com",
+        "profile_image": "https://res.cloudinary.com/practicaldev/image/fetch/s--Y1sq1tFG--/c_fill,f_auto,fl_progressive,h_640,q_auto,w_640/https://thepracticaldev.s3.amazonaws.com/uploads/user/profile_image/1/f451a206-11c8-4e3d-8936-143d0a7e65bb.png",
+        "profile_image_90": "https://res.cloudinary.com/practicaldev/image/fetch/s--DcW51A6v--/c_fill,f_auto,fl_progressive,h_90,q_auto,w_90/https://thepracticaldev.s3.amazonaws.com/uploads/user/profile_image/1/f451a206-11c8-4e3d-8936-143d0a7e65bb.png"
+    }
+}
+
+```
+
+
+### Login Functionality
+
+Login is done with [next-auth](https://next-auth.js.org) and has the following providers implemented: GitHub and Google. To check if someone is signed in, the useSession() hook is implemted and to save the state between pages, `Provider` from next-auth/client is implemented. If the application is ready for deployment, change the `NEXTAUTH_URL` in .env to the canonical URL of the website.
```

Comment:
will need to setup a .env.development and .env.production for easy of tracking and changing

Reply:
this to be discussed tonight